apm-agent-core/src/main/java/co/elastic/apm/agent/logging/Log4jLoggerFactoryBridge.java

```diff
@@ -33,6 +35,13 @@
     private static final String FQCN = Log4jLoggerFactoryBridge.class.getName();
     private static final String PACKAGE = "co.elastic.apm.agent.sdk.logging";
 
+    public static void shutdown() {
```

Comment:
Make this as a `co.elastic.apm.agent.context.LifecycleListener#stop` implementation

Reply:
Done